### PR TITLE
Remove bundlegroups when their entry bundles are removed

### PR DESCRIPTION
**Pull request by wbinnssmith**
This removes BundleGroups when their main bundle is removed from the graph.

Test Plan:
* [ ] Integration test for when a bundlegroup-entry bundle is removed when it has siblings in its groups (filed as #5143)
* [x] `yarn test`

### CODE REVIEW

File: packages/core/core/src/BundleGraph.js [hunk 1]
```diff
@@ -348,22 +348,40 @@ export default class BundleGraph {
...
-          this._graph.removeNode(bundleGroupNode);
-        } else {
-          let index = bundleGroupNode.value.bundleIds.indexOf(bundle.id);
-          invariant(index >= 0);
-          bundleGroupNode.value.bundleIds.splice(index, 1);
-        }
+      if (
+        // If the bundle group's entry asset belongs to this bundle, the group
+        // was created because of this bundle. Delete the group.
+        isBundleGroupEntry ||
```
Comment (wbinnssmith) [inline packages/core/core/src/BundleGraph.js:377]: @devongovett is this safe?

Removing the bundle node should remove any of its children, and removing the group should remove its siblings from the group. Neither of these cases should leave us in a state where we're removing access to assets that are depended on by other bundles, right?

Comment (wbinnssmith) [reply packages/core/core/src/BundleGraph.js:377]: Also, should we recursively remove child bundles that aren't reachable by any others? We currently only ever call this when deduplicating, in which case these bundles _would_ be reachable, but there's nothing stopping this from being called in other cases. cc @padmaia @devongovett

File: packages/core/core/src/BundleGraph.js [hunk 2]
```diff
@@ -348,22 +348,40 @@ export default class BundleGraph {
...
+
+  removeBundle(bundle: Bundle) {
+    // Remove bundle node if it no longer has any entry assets
+    let bundleNode = nullthrows(this._graph.getNode(bundle.id));
+
+    let bundleGroupNodes = this._graph.getNodesConnectedTo(
+      bundleNode,
+      'bundle',
+    );
+    this._graph.removeNode(bundleNode);
```
Comment (wbinnssmith) [inline packages/core/core/src/BundleGraph.js:365]: Should we recursively remove child bundles that aren't reachable by any others? We currently only ever call this when deduplicating, in which case these bundles would be reachable, but there's nothing stopping this from being called in other cases. cc @padmaia @devongovett

**Comment by parcel-benchmark** [comment]
## Benchmark Results
<details><summary>Kitchen Sink ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 8.64s | -94.00ms |
| Cached | 1.65s | +9.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 57.00ms | -19.00ms 🚀 |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 57.00ms | -20.00ms 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 45.00ms | +18.00ms ⚠️ |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 44.00ms | +17.00ms ⚠️ |
| dist/legacy/index.45cccf70.js | 1.10kb | +0.00b | 142.00ms | +19.00ms ⚠️ |
| dist/modern/index.1964d1f6.js | 1.10kb | +0.00b | 141.00ms | +19.00ms ⚠️ |
| dist/legacy/index.html | 701.00b | +0.00b | 156.00ms | +23.00ms ⚠️ |
| dist/modern/index.html | 701.00b | +0.00b | 141.00ms | +20.00ms ⚠️ |
| dist/legacy/index.b3acc8eb.css | 77.00b | +0.00b | 142.00ms | +20.00ms ⚠️ |
| dist/modern/index.7bc22a8b.css | 77.00b | +0.00b | 141.00ms | +20.00ms ⚠️ |

</p></details>

<details><summary>React HackerNews ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 34.05s | -32.00ms |
| Cached | 1.86s | +103.00ms ⚠️ |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/logo.24c8bf9e.png | 274.00b | +0.00b | 52.00ms | +18.00ms ⚠️ |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.js | 478.68kb | +0.00b | 91.00ms | -37.00ms 🚀 |
| dist/PermalinkedComment.9c5ad065.js | 4.22kb | +0.00b | 63.00ms | -35.00ms 🚀 |
| dist/UserProfile.c1927888.js | 1.62kb | +0.00b | 63.00ms | -35.00ms 🚀 |
| dist/NotFound.f8a0616d.js | 387.00b | +0.00b | 63.00ms | -35.00ms 🚀 |
| dist/logo.24c8bf9e.png | 274.00b | +0.00b | 20.00ms | +5.00ms ⚠️ |

</p></details>

<details><summary>AtlasKit Editor ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 3.13m | -9.73s 🚀 |
| Cached | 4.17s | -865.00ms 🚀 |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/pdfRenderer.15a6a023.js | 1.11mb | +0.00b | 52.12s | -9.66s 🚀 |
| dist/editorView.e06544fb.js | 587.12kb | +0.00b | 1.37m | -10.47s 🚀 |
| dist/popup.6c453e3c.js | 169.96kb | +0.00b | 52.11s | -9.70s 🚀 |
| dist/Toolbar.83ac0b6e.js | 92.34kb | +0.00b | 57.17s | -7.61s 🚀 |
| dist/EmojiPickerComponent.ae601247.js | 75.29kb | +0.00b | 1.01m | +48.88s ⚠️ |
| dist/media-viewer.bcba38b3.js | 71.96kb | +0.00b | 13.08s | -4.59s 🚀 |
| dist/card.645a5c17.js | 55.01kb | +0.00b | 10.49s | -4.30s 🚀 |
| dist/card.0313bac8.js | 52.78kb | +0.00b | 53.70s | -48.39s 🚀 |
| dist/Modal.19ffed32.js | 41.27kb | +0.00b | 6.61s | -787.00ms 🚀 |
| dist/component.54080a26.js | 22.50kb | +0.00b | 6.61s | -789.00ms 🚀 |
| dist/DatePicker.0df05dac.js | 21.21kb | +0.00b | 1.01m | +48.88s ⚠️ |
| dist/EmojiPickerComponent.7dc0ebd0.js | 18.23kb | +0.00b | 1.01m | +48.87s ⚠️ |
| dist/dropzone.7c704bdb.js | 17.27kb | +0.00b | 53.70s | -47.91s 🚀 |
| dist/smartMediaEditor.9623570c.js | 16.87kb | +0.00b | 55.88s | -7.69s 🚀 |
| dist/js.5733a017.js | 16.40kb | +0.00b | 6.61s | -788.00ms 🚀 |
| dist/ui.fee77548.js | 13.97kb | +0.00b | 10.46s | -4.32s 🚀 |
| dist/workerHasher.1e706e77.js | 11.75kb | +0.00b | 10.46s | -4.32s 🚀 |
| dist/EmojiTypeAheadComponent.1990d771.js | 8.67kb | +0.00b | 8.27s | -4.14s 🚀 |
| dist/component.77c0f682.js | 6.05kb | +0.00b | 6.36s | -835.00ms 🚀 |
| dist/card.383bd9bd.js | 5.61kb | +0.00b | 12.89s | -4.78s 🚀 |
| dist/media-viewer.59c17dad.js | 3.92kb | +0.00b | 12.94s | -4.74s 🚀 |
| dist/EmojiPickerComponent.627aca6b.js | 3.58kb | +0.00b | 8.27s | -4.14s 🚀 |
| dist/index.ac562c80.css | 3.46kb | +0.00b | 58.19s | -7.75s 🚀 |
| dist/png-chunks-extract.a844fdb6.js | 3.41kb | +0.00b | 6.61s | -789.00ms 🚀 |
| dist/dropzone.3c67f0a6.js | 3.23kb | +0.00b | 28.19s | -6.97s 🚀 |
| dist/Modal.0304b9ed.js | 3.08kb | +0.00b | 6.61s | -787.00ms 🚀 |
| dist/clipboard.da112b6e.js | 2.75kb | +0.00b | 52.98s | -9.11s 🚀 |
| dist/EmojiUploadComponent.d54a1fc6.js | 2.18kb | +0.00b | 8.27s | -4.14s 🚀 |
| dist/card.eea6ad8c.js | 2.03kb | +0.00b | 28.18s | -6.99s 🚀 |
| dist/ResourcedEmojiComponent.abe8596c.js | 2.00kb | +0.00b | 5.14s | -1.01s 🚀 |
| dist/date.231500d2.js | 1.73kb | +0.00b | 7.13s | -857.00ms 🚀 |
| dist/16.d16f14ed.js | 1.73kb | +0.00b | 6.99s | -823.00ms 🚀 |
| dist/images.80c92f5e.js | 1.67kb | +0.00b | 7.23s | -905.00ms 🚀 |
| dist/16.1b6ca79f.js | 1.66kb | +0.00b | 7.64s | -779.00ms 🚀 |
| dist/feedback.d5590b34.js | 1.64kb | +0.00b | 7.69s | -4.16s 🚀 |
| dist/browser.75813db0.js | 1.61kb | +0.00b | 53.00s | -9.11s 🚀 |
| dist/workerHasher.6d5b22d7.js | 1.54kb | +0.00b | 8.27s | -4.14s 🚀 |
| dist/workerHasher.fa198233.js | 1.54kb | +0.00b | 54.46s | -47.62s 🚀 |
| dist/list-number.c35f9589.js | 1.46kb | +0.00b | 7.37s | -917.00ms 🚀 |
| dist/status.8ab4226b.js | 1.46kb | +0.00b | 7.56s | -954.00ms 🚀 |
| dist/16.9ccb5b28.js | 1.38kb | +0.00b | 6.80s | -797.00ms 🚀 |
| dist/code.3dd71280.js | 1.38kb | +0.00b | 7.13s | -857.00ms 🚀 |
| dist/16.19c93285.js | 1.33kb | +0.00b | 7.64s | -779.00ms 🚀 |
| dist/16.c7751e34.js | 1.33kb | +0.00b | 6.80s | -797.00ms 🚀 |
| dist/16.18344f47.js | 1.31kb | +0.00b | 7.13s | -858.00ms 🚀 |
| dist/link.2f4427e6.js | 1.30kb | +0.00b | 7.37s | -917.00ms 🚀 |
| dist/heading6.fdbbdf4d.js | 1.30kb | +0.00b | 7.69s | -48.42s 🚀 |
| dist/heading3.2113b7a1.js | 1.29kb | +0.00b | 7.56s | -959.00ms 🚀 |
| dist/16.2c4c1846.js | 1.27kb | +0.00b | 6.99s | -824.00ms 🚀 |
| dist/emoji.93ae61d3.js | 1.23kb | +0.00b | 7.23s | -905.00ms 🚀 |
| dist/16.d17ab186.js | 1.23kb | +0.00b | 6.99s | -823.00ms 🚀 |
| dist/16.b4da0acb.js | 1.19kb | +0.00b | 6.99s | -823.00ms 🚀 |
| dist/16.02c68c4f.js | 1.18kb | +0.00b | 7.13s | -857.00ms 🚀 |
| dist/heading5.e195aab1.js | 1.17kb | +0.00b | 7.69s | -48.42s 🚀 |
| dist/expand.450f3aea.js | 1.16kb | +0.00b | 7.69s | -4.16s 🚀 |
| dist/media-card-analytics-error-boundary.2323af57.js | 1.15kb | +0.00b | 28.18s | -6.99s 🚀 |
| dist/16.4e3b0235.js | 1.14kb | +0.00b | 6.80s | -797.00ms 🚀 |
| dist/16.a37df53b.js | 1.13kb | +0.00b | 6.80s | -797.00ms 🚀 |
| dist/16.9adb4ec7.js | 1.13kb | +0.00b | 6.80s | -797.00ms 🚀 |
| dist/16.eafe1a11.js | 1.13kb | +0.00b | 7.64s | -778.00ms 🚀 |
| dist/heading2.a2a1be3f.js | 1.11kb | +0.00b | 7.56s | -954.00ms 🚀 |
| dist/16.3920766f.js | 1.10kb | +0.00b | 5.14s | -1.01s 🚀 |
| dist/component.6600f83e.js | 1.07kb | +0.00b | 5.14s | -1.01s 🚀 |
| dist/mention.4d64e059.js | 1.07kb | +0.00b | 7.37s | -917.00ms 🚀 |
| dist/heading4.ec980e93.js | 1.06kb | +0.00b | 7.69s | -48.42s 🚀 |
| dist/16.afcb09b6.js | 1.05kb | +0.00b | 6.99s | -823.00ms 🚀 |
| dist/layout.52ba15f7.js | 1.05kb | +0.00b | 7.23s | -905.00ms 🚀 |
| dist/divider.16defde6.js | 1.03kb | +0.00b | 7.23s | -905.00ms 🚀 |
| dist/quote.aa25b83c.js | 1.02kb | +0.00b | 7.47s | -912.00ms 🚀 |
| dist/action.9aa72b39.js | 1.00kb | +0.00b | 7.13s | -856.00ms 🚀 |
| dist/decision.20244046.js | 1010.00b | +0.00b | 7.23s | -906.00ms 🚀 |
| dist/panel-warning.97a0e3b3.js | 1008.00b | +0.00b | 7.47s | -912.00ms 🚀 |
| dist/list.c9147a62.js | 978.00b | +0.00b | 7.37s | -917.00ms 🚀 |
| dist/heading1.7d899ba5.js | 976.00b | +0.00b | 7.56s | -954.00ms 🚀 |
| dist/panel-error.8d11e8b7.js | 906.00b | +0.00b | 7.37s | -917.00ms 🚀 |
| dist/panel.a28c56ee.js | 897.00b | +0.00b | 7.47s | -912.00ms 🚀 |
| dist/table.5a4874ee.js | 889.00b | +0.00b | 7.56s | -954.00ms 🚀 |
| dist/panel-success.9b923e58.js | 845.00b | +0.00b | 7.47s | -912.00ms 🚀 |
| dist/panel-note.97a4d541.js | 841.00b | +0.00b | 7.47s | -912.00ms 🚀 |
| dist/media-picker-analytics-error-boundary.f8ddfee3.js | 839.00b | +0.00b | 52.98s | -9.12s 🚀 |
| dist/media-card-analytics-error-boundary.663369b9.js | 835.00b | +0.00b | 10.46s | -4.32s 🚀 |
| dist/media-card-analytics-error-boundary.73ba4074.js | 835.00b | +0.00b | 54.46s | -47.62s 🚀 |
| dist/media-viewer-analytics-error-boundary.0afa7031.js | 755.00b | +0.00b | 13.03s | -4.65s 🚀 |
| dist/EmojiPickerComponent.2e54b9c6.js | 535.00b | +0.00b | 8.27s | -4.14s 🚀 |
| dist/simpleHasher.16c8bf41.js | 528.00b | +0.00b | 10.46s | -4.32s 🚀 |
| dist/simpleHasher.a29b2a8e.js | 528.00b | +0.00b | 54.49s | -8.87s 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.975a5c74.js | 2.72mb | +0.00b | 184.00ms | -17.00ms 🚀 |
| dist/pdfRenderer.15a6a023.js | 1.11mb | +0.00b | 182.00ms | -16.00ms 🚀 |
| dist/editorView.e06544fb.js | 587.12kb | +0.00b | 181.00ms | -16.00ms 🚀 |
| dist/popup.6c453e3c.js | 169.96kb | +0.00b | 181.00ms | -16.00ms 🚀 |
| dist/Toolbar.83ac0b6e.js | 92.34kb | +0.00b | 181.00ms | -15.00ms 🚀 |
| dist/EmojiPickerComponent.ae601247.js | 75.29kb | +0.00b | 181.00ms | -18.00ms 🚀 |
| dist/media-viewer.bcba38b3.js | 71.96kb | +0.00b | 182.00ms | -16.00ms 🚀 |
| dist/card.645a5c17.js | 55.01kb | +0.00b | 182.00ms | -16.00ms 🚀 |
| dist/card.0313bac8.js | 52.78kb | +0.00b | 181.00ms | -16.00ms 🚀 |
| dist/Modal.19ffed32.js | 41.27kb | +0.00b | 184.00ms | -16.00ms 🚀 |
| dist/component.d7c29980.js | 30.70kb | +0.00b | 184.00ms | -16.00ms 🚀 |
| dist/component.54080a26.js | 22.50kb | +0.00b | 184.00ms | -16.00ms 🚀 |
| dist/DatePicker.0df05dac.js | 21.21kb | +0.00b | 181.00ms | -18.00ms 🚀 |
| dist/EmojiPickerComponent.7dc0ebd0.js | 18.23kb | +0.00b | 181.00ms | -18.00ms 🚀 |
| dist/dropzone.7c704bdb.js | 17.27kb | +0.00b | 181.00ms | -16.00ms 🚀 |
| dist/smartMediaEditor.9623570c.js | 16.87kb | +0.00b | 181.00ms | -16.00ms 🚀 |
| dist/js.5733a017.js | 16.40kb | +0.00b | 184.00ms | -16.00ms 🚀 |
| dist/ui.fee77548.js | 13.97kb | +0.00b | 182.00ms | -16.00ms 🚀 |
| dist/workerHasher.1e706e77.js | 11.75kb | +0.00b | 181.00ms | -17.00ms 🚀 |
| dist/EmojiTypeAheadComponent.1990d771.js | 8.67kb | +0.00b | 181.00ms | -17.00ms 🚀 |
| dist/component.77c0f682.js | 6.05kb | +0.00b | 184.00ms | -16.00ms 🚀 |
| dist/card.383bd9bd.js | 5.61kb | +0.00b | 182.00ms | -16.00ms 🚀 |
| dist/media-viewer.59c17dad.js | 3.92kb | +0.00b | 181.00ms | -17.00ms 🚀 |
| dist/EmojiPickerComponent.627aca6b.js | 3.58kb | +0.00b | 181.00ms | -17.00ms 🚀 |
| dist/index.ac562c80.css | 3.46kb | +0.00b | 181.00ms | -15.00ms 🚀 |
| dist/png-chunks-extract.a844fdb6.js | 3.41kb | +0.00b | 184.00ms | -16.00ms 🚀 |
| dist/dropzone.3c67f0a6.js | 3.23kb | +0.00b | 181.00ms | -16.00ms 🚀 |
| dist/Modal.0304b9ed.js | 3.08kb | +0.00b | 184.00ms | -16.00ms 🚀 |
| dist/clipboard.da112b6e.js | 2.75kb | +0.00b | 181.00ms | -16.00ms 🚀 |
| dist/16.290b4a43.js | 2.38kb | +0.00b | 184.00ms | -16.00ms 🚀 |
| dist/EmojiUploadComponent.d54a1fc6.js | 2.18kb | +0.00b | 181.00ms | -17.00ms 🚀 |
| dist/card.eea6ad8c.js | 2.03kb | +0.00b | 181.00ms | -16.00ms 🚀 |
| dist/ResourcedEmojiComponent.abe8596c.js | 2.00kb | +0.00b | 184.00ms | -17.00ms 🚀 |
| dist/date.231500d2.js | 1.73kb | +0.00b | 183.00ms | -16.00ms 🚀 |
| dist/16.d16f14ed.js | 1.73kb | +0.00b | 183.00ms | -17.00ms 🚀 |
| dist/images.80c92f5e.js | 1.67kb | +0.00b | 183.00ms | -17.00ms 🚀 |
| dist/16.1b6ca79f.js | 1.66kb | +0.00b | 183.00ms | -17.00ms 🚀 |
| dist/feedback.d5590b34.js | 1.64kb | +0.00b | 181.00ms | -18.00ms 🚀 |
| dist/browser.75813db0.js | 1.61kb | +0.00b | 181.00ms | -16.00ms 🚀 |
| dist/workerHasher.6d5b22d7.js | 1.54kb | +0.00b | 181.00ms | -17.00ms 🚀 |
| dist/workerHasher.fa198233.js | 1.54kb | +0.00b | 181.00ms | -16.00ms 🚀 |
| dist/list-number.c35f9589.js | 1.46kb | +0.00b | 182.00ms | -17.00ms 🚀 |
| dist/status.8ab4226b.js | 1.46kb | +0.00b | 181.00ms | -18.00ms 🚀 |
| dist/16.9ccb5b28.js | 1.38kb | +0.00b | 183.00ms | -17.00ms 🚀 |
| dist/code.3dd71280.js | 1.38kb | +0.00b | 183.00ms | -16.00ms 🚀 |
| dist/16.19c93285.js | 1.33kb | +0.00b | 183.00ms | -17.00ms 🚀 |
| dist/16.c7751e34.js | 1.33kb | +0.00b | 183.00ms | -17.00ms 🚀 |
| dist/16.51049e2c.js | 1.32kb | +0.00b | 184.00ms | -16.00ms 🚀 |
| dist/16.18344f47.js | 1.31kb | +0.00b | 183.00ms | -16.00ms 🚀 |
| dist/link.2f4427e6.js | 1.30kb | +0.00b | 182.00ms | -17.00ms 🚀 |
| dist/heading6.fdbbdf4d.js | 1.30kb | +0.00b | 181.00ms | -18.00ms 🚀 |
| dist/heading3.2113b7a1.js | 1.29kb | +0.00b | 181.00ms | -18.00ms 🚀 |
| dist/16.2c4c1846.js | 1.27kb | +0.00b | 183.00ms | -16.00ms 🚀 |
| dist/emoji.93ae61d3.js | 1.23kb | +0.00b | 183.00ms | -17.00ms 🚀 |
| dist/16.d17ab186.js | 1.23kb | +0.00b | 183.00ms | -17.00ms 🚀 |
| dist/16.436f097f.js | 1.21kb | +0.00b | 184.00ms | -16.00ms 🚀 |
| dist/16.b4da0acb.js | 1.19kb | +0.00b | 183.00ms | -16.00ms 🚀 |
| dist/16.02c68c4f.js | 1.18kb | +0.00b | 183.00ms | -16.00ms 🚀 |
| dist/heading5.e195aab1.js | 1.17kb | +0.00b | 181.00ms | -18.00ms 🚀 |
| dist/expand.450f3aea.js | 1.16kb | +0.00b | 181.00ms | -18.00ms 🚀 |
| dist/media-card-analytics-error-boundary.2323af57.js | 1.15kb | +0.00b | 181.00ms | -16.00ms 🚀 |
| dist/16.b46ef690.js | 1.14kb | +0.00b | 184.00ms | -16.00ms 🚀 |
| dist/16.4e3b0235.js | 1.14kb | +0.00b | 183.00ms | -17.00ms 🚀 |
| dist/16.9ba230ea.js | 1.13kb | +0.00b | 183.00ms | -17.00ms 🚀 |
| dist/16.a37df53b.js | 1.13kb | +0.00b | 183.00ms | -17.00ms 🚀 |
| dist/16.9adb4ec7.js | 1.13kb | +0.00b | 183.00ms | -17.00ms 🚀 |
| dist/16.eafe1a11.js | 1.13kb | +0.00b | 183.00ms | -17.00ms 🚀 |
| dist/heading2.a2a1be3f.js | 1.11kb | +0.00b | 181.00ms | -18.00ms 🚀 |
| dist/16.3920766f.js | 1.10kb | +0.00b | 184.00ms | -16.00ms 🚀 |
| dist/component.6600f83e.js | 1.07kb | +0.00b | 184.00ms | -17.00ms 🚀 |
| dist/mention.4d64e059.js | 1.07kb | +0.00b | 182.00ms | -17.00ms 🚀 |
| dist/heading4.ec980e93.js | 1.06kb | +0.00b | 181.00ms | -18.00ms 🚀 |
| dist/16.afcb09b6.js | 1.05kb | +0.00b | 183.00ms | -16.00ms 🚀 |
| dist/layout.52ba15f7.js | 1.05kb | +0.00b | 183.00ms | -16.00ms 🚀 |
| dist/divider.16defde6.js | 1.03kb | +0.00b | 183.00ms | -16.00ms 🚀 |
| dist/quote.aa25b83c.js | 1.02kb | +0.00b | 181.00ms | -18.00ms 🚀 |
| dist/action.9aa72b39.js | 1.00kb | +0.00b | 183.00ms | -16.00ms 🚀 |
| dist/decision.20244046.js | 1010.00b | +0.00b | 183.00ms | -16.00ms 🚀 |
| dist/panel-warning.97a0e3b3.js | 1008.00b | +0.00b | 182.00ms | -17.00ms 🚀 |
| dist/list.c9147a62.js | 978.00b | +0.00b | 182.00ms | -17.00ms 🚀 |
| dist/heading1.7d899ba5.js | 976.00b | +0.00b | 181.00ms | -18.00ms 🚀 |
| dist/panel-error.8d11e8b7.js | 906.00b | +0.00b | 182.00ms | -17.00ms 🚀 |
| dist/panel.a28c56ee.js | 897.00b | +0.00b | 181.00ms | -18.00ms 🚀 |
| dist/table.5a4874ee.js | 889.00b | +0.00b | 181.00ms | -18.00ms 🚀 |
| dist/panel-success.9b923e58.js | 845.00b | +0.00b | 182.00ms | -17.00ms 🚀 |
| dist/panel-note.97a4d541.js | 841.00b | +0.00b | 182.00ms | -17.00ms 🚀 |
| dist/media-picker-analytics-error-boundary.f8ddfee3.js | 839.00b | +0.00b | 181.00ms | -16.00ms 🚀 |
| dist/media-card-analytics-error-boundary.663369b9.js | 835.00b | +0.00b | 182.00ms | -16.00ms 🚀 |
| dist/media-card-analytics-error-boundary.73ba4074.js | 835.00b | +0.00b | 181.00ms | -16.00ms 🚀 |
| dist/media-viewer-analytics-error-boundary.0afa7031.js | 755.00b | +0.00b | 181.00ms | -16.00ms 🚀 |
| dist/EmojiPickerComponent.2e54b9c6.js | 535.00b | +0.00b | 181.00ms | -17.00ms 🚀 |
| dist/simpleHasher.16c8bf41.js | 528.00b | +0.00b | 182.00ms | -16.00ms 🚀 |
| dist/simpleHasher.a29b2a8e.js | 528.00b | +0.00b | 181.00ms | -16.00ms 🚀 |
| dist/index.html | 119.00b | +0.00b | 205.00ms | -22.00ms 🚀 |

</p></details>

<details><summary>Three.js x4 🚨</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | **FAILED** | -0.00ms |
| Cached | **FAILED** | -0.00ms |

#### Cold Bundles

*No bundles found, this is probably a failed build...*
#### Cached Bundles

*No bundles found, this is probably a failed build...*
</p></details>

**[Click here to view a detailed benchmark overview.](https://benchmark.parceljs.org/benchmark/276689087534989836)**

File: packages/core/core/src/BundleGraph.js (hunk 2 above)
Comment (devongovett) [reply packages/core/core/src/BundleGraph.js:366]: Not yet in the public API though right? So plugins wouldn't have access to this.

**Review by devongovett — APPROVED** [review]

**Pull request merged**